app/assets/javascripts/components/PipelineSampleReads.jsx

```diff
@@ -18,14 +18,64 @@ class PipelineSampleReads extends React.Component {
     this.summary_stats = props.summary_stats;
     this.gotoReport = this.gotoReport.bind(this);
     this.sampleId = this.sampleInfo.id;
+    this.host_genome = props.host_genome;
     this.pipelineStatus = props.sample_status
     this.pipelineRun = props.pipelineRun
     this.rerunPipeline = this.rerunPipeline.bind(this);
     this.state = {
       rerun: false,
       failureText: 'Sample run failed'
     };
-    this.TYPE_PROMPT = "Type here..."
+    this.TYPE_PROMPT = "Type here...";
+    this.TISSUE_TYPES = ["Bronchoalveolar lavage", "Cerebrospinal fluid",
+                         "Nasopharyngeal swab", "Plasma", "Serum", "Solid tissue", 
+                         "Stool", "Synovial fluid", "Whole blood", "Other"];
+    this.NUCLEOTIDE_TYPES = ["DNA", "RNA"];
+    this.DROPDOWN_OPTIONS = { sample_tissue: this.TISSUE_TYPES,
+                              sample_template: this.NUCLEOTIDE_TYPES };
+    this.DROPDOWN_METADATA_FIELDS = Object.keys(this.DROPDOWN_OPTIONS);
+    this.handleDropdownChange = this.handleDropdownChange.bind(this);
+  }
+
+  render_metadata_dropdown(label, field) {
+    dropdown_options = this.DROPDOWN_OPTIONS[field]
```

Comment:
Looks good but we should make these local variables instead, prefix with either const or let eg
`let dropdown_option = <some_value>`
same comment for line 42, 59, 60, 61, 124